### ISSUE TITLE
groupBy not working

### ISSUE THREAD

**Issue by ahmedgaafer** [srigar/multiselect-react-dropdown#179]
When trying to add groupBy options to the component I get the following error

![image](https://user-images.githubusercontent.com/23239056/146778052-7b8f7fdc-8c90-413a-b22c-00b7e3c497fe.png)

the same error shows in the docs 

my code

```js

let options = [
			{ name: 'Option 1', id: 1, g: 'g1'},
			{ name: 'Option 2', id: 2, g: 'g1' },
			{ name: 'Option 3', id: 3, g: 'g2' },
			{ name: 'Option 4', id: 4, g: 'g2' },
		];


<Multiselect
  options={this.state.options} // Options to display in the dropdown
  displayValue="name" // Property name to display in the dropdown options
  showCheckbox={true} // Display a checkbox next to each option
  groupBy="g" // Property to group options by
  />
```

**Comment by AjXUthaya** [comment]
Demo not working either: https://multiselect-react-dropdown.vercel.app/?path=/story/multiselect-dropdown--grouping

**Comment by AjXUthaya** [comment]
Rollback and use multiselect-react-dropdown@2.0.9, it seems to be working from what i can see

**Comment by EmirSoza** [comment]
(To the developer) : It seems like there is an unknown value (option) is used on renderGroupByOptions function instead of using obj.
return Object.keys(groupedObject).map(function (obj) {
        var isSelected = _this6.isSelectedValue(option); 
        ......

**Comment by srigar** [comment]
Fix available in v2.0.20
